_posts/2014-01-01-callback.md

```diff
@@ -1,4 +1,5 @@
 ---
+title: Callback
```

Comment:
Probably, I'm just trying to keep the YAML front matter consistent. As a rule, I try to keep any formatting consistent across all members of set so that someone who wants to create a new member can pick one at random to emulate and not diverge from the norm. In essence, everything is a "template".